src/renderer/components/watch-video-description/watch-video-description.js

```diff
@@ -31,13 +30,21 @@ export default Vue.extend({
   },
   mounted: function () {
     if (this.descriptionHtml !== '') {
-      this.shownDescription = this.parseDescriptionHtml(this.descriptionHtml)
-    } else {
-      this.shownDescription = autolinker.link(this.description)
-    }
+      const parsed = this.parseDescriptionHtml(this.descriptionHtml)
+
+      // the invidious API returns emtpy html elements when the description is empty
+      // so we need to check for that here
 
-    if (/^\s*$/.test(this.shownDescription)) {
-      $('.videoDescription')[0].style.display = 'none'
+      const testDiv = document.createElement('div')
```

Comment:
I'm guessing this should be removed?

Reply:
No, the purpose of the test div is to check if there is any text in the descriptionHtml other than just HTML elements. I could just check for `<p></p>` but as Invidious might change in the future I wanted a more generic solution.
We are letting the browser parse the HTML by assigning it to the `innerHTML` property of the testDiv. When descriptionHtml is just `<p></p>` the `innerText` property of the testDiv is an empty string but when there is text in descriptionHtml e.g. `<p>text</p>` then the `innerText` property of the testDiv is `text`. So by checking the length of the `innerText` property of the testDiv we know whether the description is empty or not.

Reply:
Added a comment about the HTML parsing to the code now.

Reply:
If that's needed, we could make the API return an empty string.
I'd prefer for us to return clean and consistent data, rather than requiring hacky workarounds downstream!

Reply:
That would be very much appreciated, as long is it doesn't break stuff for other people using the API.